app/controllers/problem_reports_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class ProblemReportsController < ApplicationController
+  def index
+    @problem_reports = ProblemReport.order(id: :desc)
+    filter_encode
+
+    if params[:filter] == 'Unfixed and Critical'
+      @problem_reports.where!(is_critical: true, is_fixed: false)
```

Comment:
Use `where`, `where!` methods are not meant to be used.

https://stackoverflow.com/questions/23950627/undocumented-activerecord-bang-methods